backend/runprocessors.py

```diff
@@ -10,13 +10,13 @@
 from algorithms.load_conditions import cache_conditions_pkl_file
 from algorithms.log_broken import log_broken_conditions 
 
-from data.scrapers.programsScraper import scrape_programs as scrape_prg_data
-from data.scrapers.specialisationsScraper import scrape_spn_data
-from data.scrapers.coursesScraper import scrape_courses as scrape_course_data
```

Comment:
Maybe can standardise the function names as well inside all these files instead of us having to rename the function in the import